src/fulfillment/Dockerfile.jvm

```diff
@@ -0,0 +1,43 @@
+ARG APPLICATION_NAME="fulfillment"
```

Comment:
what's the reason for the alternate Dockerfile?

Reply:
This builds it as a normal micronaut app running on a JVM

Reply:
Can be useful to build both and compare native vs JVM